src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1148,12 +1148,58 @@ private function registerAnnotationsConfiguration(array $config, ContainerBuilde
         }
     }
 
-    private function registerPropertyAccessConfiguration(array $config, ContainerBuilder $container)
+    private function registerPropertyAccessConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader)
     {
+        $loader->load('property_access.xml');
+
+        $chainLoader = $container->getDefinition('property_access.mapping.chain_loader');
+
+        $serializerLoaders = array();
+        if (isset($config['enable_annotations']) && $config['enable_annotations']) {
+            if (!class_exists('Doctrine\Common\Annotations\Annotation')) {
```

Comment:
We prefer the `::class` constant for all new code (same for all class names below)